css/style.css

```diff
@@ -37,23 +37,188 @@ Developer @GA in NYC
   float: left;
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
+.categories {
+  display: block;
+  font-weight:300;
+  font-size:24px;
+}
+.nav li {
+  display: inline-block;
+  color: #f9e42e;
+  font-weight: 600;
+  font-size: 32px;
+  padding: 8px;
+}
+.nav ul {
+  list-style: none;
+  overflow: auto;
+  width: 600px;
+  margin: 0 auto;
+
+}
+.signup {
+  float: left;
+  width: 100%;
+  background: #f9e42e;
+  margin: auto;
+  padding: 50px;
+}
+
+.SignUpB {
+  background-color: #033048;
+  font-weight: 800;
+  font-size: 24px;
+  border-radius: 5px;
+  padding: 10px 35px;
+  margin: 5px auto;
+  display: block;
+  border: none;
+  color: white;
+}
+.social{ 
+  margin: auto;
+  text-align: center;
+}
+.read{
+  float: right;
+  width: 25%;
+  text-decoration: none;
+  font-size: 22px;
+  font-weight: 600;
+  color: #033048;
+  padding: 15px;
+}
+.about {
+  padding: 20px;
+}
 
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
-body {
+main {
```

Comment:
add a CSS rule that sets all images to have a `max-width: 100%`. This way your images inside the `<main>` tag won't expand beyond the 700px width